instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from sqlalchemy.event import listen
+from sqlalchemy.event import listen  # pylint: disable=no-name-in-module
```

Comment:
It's an issue with latest pylint. https://github.com/PyCQA/pylint/issues/2648 and https://github.com/PyCQA/pylint/issues/2862

Let's discuss in the SIG meeting this week. We can either revert to older version of pylint for contrib repo or we can rename instrumentation packages to something different from the libraries they are instrumenting. For example, instrumentation for requests could be called `requests_otel` or `requests_instrumentation` instead of `requests`.

Reply:
Resolving. Let's discuss in SIG. Happy to come back to fix however we decide in SIG.